Sources/NIO/BaseSocket.swift

```diff
@@ -12,6 +12,28 @@
 //
 //===----------------------------------------------------------------------===//
 
+import NIOConcurrencyHelpers
+
+private let globallyIgnoredSIGPIPE: Atomic<Bool> = .init(value: false)
+
+func ignoreSIGPIPE(descriptor fd: CInt) throws {
```

Comment:
Is there any reason for this function not to be namespaced into `BaseSocket`?

Reply:
done (to `BaseSocketProtocol`)